articles/sql-or-nosql-when-to-choose-what/index.md

```diff
@@ -0,0 +1,123 @@
+
+It is not like the early times when data was stored on papers and cabinet. Data is now all around us. Information is the backbone of almost every modern business models. With the current steep growth of the database technology, as a developer building an application to store some data online, and already know what a database is, the big question you should be asking yourself is which database model will fit your application model. Knowing which database structure to use is critical to building outstanding applications.
+SQL or NoSQL? That is what we will dive in briefly, to understand their structures, explore them deeply, and understand how they work. These two databases do one thing, to store data, but the approach to do this may differ.
```

Comment:
**These two databases do one thing, to store data, but the approach to do this may differ.**

"SQL and NoSQL databases store data, but the approach they take differs" would flow better.